### ISSUE TITLE
Feature: Remove a setting

### ISSUE THREAD

**Issue by MGatner** [codeigniter4/settings#7]
Maybe an addendum to the previous issue, but we need a meaningful way to remove/forget an overriding setting from its handlers.

**Comment by lonnieezell** [comment]
I was thinking about this last night, also. Will do.

**Comment by MGatner** [comment]
That was fast! A little hard to read changes, looks like your IDE is waffling on tabs/spaces. 

Just to stir the pot (I already know you're going to ban me from all your future projects)... this is looking more and more like Cache. I assume we'll eventually want a Settings Redis Handler, etc. Are we reinventing? Would this be better implemented on top of the Cache layer, or something like a special case that used the Cache Handlers without its service?

**Comment by lonnieezell** [comment]
My fault on the formatting, probably. I accidentally ran `composer style` before pushing it in a different branch which affected more files than expected, so I tried to reset that and looks like I borked it. Default on my IDE is PSRE12, pretty sure, but I'll double-check. 

And, no, I don't have any plans on adding other handlers. To me the goal is to support being able to dynamically save the config, which the database is perfect for. While building it I thought maybe there would be times when other's apps might have other needs, so I made it work as a handler. That was the extent of my plans :)
